### ISSUE TITLE
Color link dosen't work with dbc.dropdownMenu

### ISSUE THREAD

**Issue by vokiatik** [facultyai/dash-bootstrap-components#879]
- dash version: `#2.5.1`
- dash-bootstrap-components version: `#1.2.0`
- components affected by bug:
dbc.dropdownMenu
### What is happening?

Color='link' doesn't work with dbc dropdownMenu when group = True when inside buttongroup inside offcanvas (don't know if it does matter)

### What should be happening?

it should work

### Code

```python
dbc.DropdownMenu([
    dbc.DropdownMenuItem("item1", href="#",
    dbc.DropdownMenuItem("item2", href="#"),
    ],
    label="label",
    color="link",
    group=True
),
```

**Comment by vokiatik** [comment]
Editional info:

`menu = dbc.Offcanvas(children=[
            dbc.ButtonGroup([
                dbc.Button(
                    "Домой",
                    href="http://10.104.4.95:8080/",
                    external_link=True,
                    color="link",
                ),
                dbc.Button(
                    "Таблицы",
                    href="http://10.104.4.95:8080/db_view",
                    external_link=True,
                    color="link",
                ),
                dbc.Button(
                    "Отчеты",
                    href="http://10.104.4.95:8080/tabs",
                    external_link=True,
                    color="link",
                ),
                dbc.DropdownMenu([
                    dbc.DropdownMenuItem("Процент удалений", href="http://10.104.4.95:8080/dashboard_setup?dashboard_metric=deletion"),
                    dbc.DropdownMenuItem("Своевременность отгрузки", href="http://10.104.4.95:8080/dashboard_setup?dashboard_metric=shipment"),
                    dbc.DropdownMenuItem("Уровень сервиса экспедиции", href="http://10.104.4.95:8080/dashboard_setup?dashboard_metric=service_level"),
                    dbc.DropdownMenuItem("Репрезентативность экспедиции", href="http://10.104.4.95:8080/dashboard_setup?dashboard_metric=representation"),
                    dbc.DropdownMenuItem("Уровень сервиса клиента", href="http://10.104.4.95:8080/dashboard_setup?dashboard_metric=client_service_level"),
                    dbc.DropdownMenuItem("Репрезентативность клиентской экспедиции", href="http://10.104.4.95:8080/dashboard_setup?dashboard_metric=client_representation"),
                    ],
                    label="Показатели",
                    color="link",
                    group=True,
                    className="btn btn-link"
                ),
            ],
            vertical=True,
            )
            ],
            id="menu",
            placement='start',
            scrollable=True,
            title="Меню",
            is_open=False,
            )`

![SCREENSHOT](https://user-images.githubusercontent.com/72739997/179499276-2ed128c7-bdc5-4383-a5fc-a6920f0d82f9.PNG)

**Comment by tcbegley** [comment]
Thanks for reporting @vokiatik, and apologies for the slow response.

I've just released version 1.2.1 which should fix this.

```
pip install -U dash-bootstrap-components
```
